README.md

```diff
@@ -219,6 +220,20 @@ await client.logon();
 await client.logoff();
 ```
 
+## refreshClient callback
+The refreshClient is an async callback function with the sdk client as parameter, it is called when the ACC session is expired.
```

Comment:
For consistence, use SDK (in uppercase) in the documentation.

Reply:
done